src/main/java/org/folio/rest/impl/VoucherStorageImpl.java

```diff
@@ -0,0 +1,74 @@
+package org.folio.rest.impl;
+
+import static org.folio.rest.persist.HelperUtils.getEntitiesCollection;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import java.util.Map;
+import javax.ws.rs.core.Response;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.Voucher;
+import org.folio.rest.jaxrs.model.VoucherCollection;
+import org.folio.rest.jaxrs.resource.InvoiceStorage.PutInvoiceStorageInvoicesByIdResponse;
```

Comment:
This import is probably not getting used anywhere - https://sonarcloud.io/code?id=org.folio%3Amod-invoice-storage&line=14&pullRequest=9&selected=org.folio%3Amod-invoice-storage%3Asrc%2Fmain%2Fjava%2Forg%2Ffolio%2Frest%2Fimpl%2FVoucherStorageImpl.java
Sonar cloud is complaining as well. Can remove it.

Reply:
done